cmd/projects.go

```diff
@@ -139,4 +139,32 @@ var (
 			_ = autocomplete.ProjectsGenericListAutoComplete(ctx)
 		},
 	}
+
+	projectsInfoCommand = cli.Command{
+		Name:      "projects-info",
+		Category:  "Projects",
+		Usage:     "Info on a project",
+		ArgsUsage: "project-id",
+		Description: CommandDescription{
+			Description: "Get info on a project",
+			Examples:    []string{"scalingo projects-info --project-id prj-00000000-0000-0000-0000-000000000000"},
```

Comment:
I don't think you need a flag to provide the project ID.
```suggestion
			Examples:    []string{"scalingo projects-info prj-00000000-0000-0000-0000-000000000000"},
```